docs/sources/setup-grafana/configure-security/configure-authentication/saml/index.md

```diff
@@ -231,21 +231,32 @@ This app registration will be used as a Service Account to retrieve more informa
 
 1. Go to the [Azure portal](https://portal.azure.com/#home) and sign in with your Azure AD account.
 1. In the left-hand navigation pane, select the Azure Active Directory service, and then select **App registrations**.
-1. Select **New registration**.
+1. Click the **New registration** button.
 1. In the **Register an application** pane, enter a name for the application.
 1. In the **Supported account types** section, select the account types that can use the application.
 1. In the **Redirect URI** section, select Web and enter `https://localhost/login/azuread`.
-1. Select **Register**.
+1. Click the **Register** button.
 
 #### Set up permissions for the application
 
 1. In the overview pane, look for **API permissions** section and select **Add a permission**.
 1. In the **Request API permissions** pane, select **Microsoft Graph**, and click **Application permissions**.
 1. In the **Select permissions** pane, under the **GroupMember** section, select **GroupMember.Read.All**.
+1. In the **Select permissions** pane, under the **User** section, select **User.Read.All**.
+1. Click the **Add permissions** button at the bottom of the page.
+1. In the **Request API permissions** pane, select **Microsoft Graph**, and click **Delegated permissions**.
 1. In the **Select permissions** pane, under the **User** section, select **User.Read**.
-1. Select **Add permissions** at the bottom of the page.
+1. Click the **Add permissions** button at the bottom of the page.
 1. In the **API permissions** section, select **Grant admin consent for <your-organization>**.
 
+| Permissions name | Type        | Admin consent required | Status  |
+| ---------------- | ----------- | ---------------------- | ------- |
+| `Group.Read.All` | Application | Yes                    | Granted |
+| `User.Read`      | Delegated   | No                     | Granted |
+| `User.Read.All`  | Application | Yes                    | Granted |
+
+{{< figure src="/media/docs/grafana/saml/graph-api-app-permissions.png" caption="Screen shot of the permissions listed in Azure AD for the App registration" >}}
```

Comment:
One question, do we want to maintain a screenshot when all the information is in the table?

Reply:
I could remove the table. I added both so users could understand what it looked like. @jdbaldry, would you like me to remove it?

Reply:
I think having both is fine as long as we maintain both. It's important for accessibility to try and use words rather than images to describe things and where we use images, we should have alt text or captions that describe the image to those that can't see them.

In this case, I would think a brief sentence before the table explaining that the table shows the permissions you would see in the Azure AD App registration would be valuable.